wttr-moon.widget/index.jsx

```diff
@@ -0,0 +1,33 @@
+//
+// Shows the current wttr.in forecast on your desktop
+//
+
+export const refreshFrequency = 1000 * 60 * 30; // 30min
+
+export const className = `
+	// position on screen
+	right: 100px;
+	top: 209px;
+
+	position: fixed;
+	-webkit-font-smoothing: antialiased; // nicer font rendering
+	color: #efefef;
+
+	pre
+		font: 7px "DejaVu Sans Mono", Menlo, "Lucida Sans Typewriter", "Lucida Console", monaco, "Bitstream Vera Sans Mono", monospace;
+
+`;
+
+export const command = `
+	cd wttr-moon.widget &&
+	curl -s wttr.in/moon?lang=en | ./terminal-to-html
+`;
+
+// see https://github.com/chubin/wttr.in for API
+// uses https://github.com/buildkite/terminal
+
+export const render = props => props.error ? props.error :
+	<div>
+		<link rel="stylesheet" href="wttr-moon.widget/terminal-colors.css" />
+		<pre dangerouslySetInnerHTML={{ __html: props.output }} />
```

Comment:
I think the `out.err || out.data.split('\n').slice(0, 24).join('\n')` was kind of important for filtering news messages by wttr.in

Reply:
Oh true, much cleaner. Fixed in d406dbabc4b9c1d51ca1fd419c5971b6f8ed5eb8